classes/Customer.php

```diff
@@ -565,7 +565,7 @@ public static function isBanned($idCustomer)
      * Check if e-mail is already registered in database.
      *
      * @param string $email e-mail
-     * @param bool $returnId
+     * @param bool $returnId Return Customer ID if true, or boolean
```

Comment:
Misplaced comment

Reply:
@FabienPapet no, this parameter ask to return an int or a boolean. I've changed to a clearer explanation